armotypes/notificationreporttypes.go

```diff
@@ -0,0 +1,151 @@
+package armotypes
+
+type WeeklyReport struct {
+	ClustersScannedThisWeek             int                 `json:"clustersScannedThisWeek" bson:"clustersScannedThisWeek"`
+	ClustersScannedPrevWeek             int                 `json:"clustersScannedPrevWeek" bson:"clustersScannedPrevWeek"`
+	LinkToConfigurationScanningFiltered string              `json:"linkToConfigurationScanningFiltered" bson:"linkToConfigurationScanningFiltered"`
+	RepositoriesScannedThisWeek         int                 `json:"repositoriesScannedThisWeek" bson:"repositoriesScannedThisWeek"`
+	RepositoriesScannedPrevWeek         int                 `json:"repositoriesScannedPrevWeek" bson:"repositoriesScannedPrevWeek"`
+	LinkToRepositoriesScanningFiltered  string              `json:"linkToRepositoriesScanningFiltered" bson:"linkToRepositoriesScanningFiltered"`
+	RegistriesScannedThisWeek           int                 `json:"registriesScannedThisWeek" bson:"registriesScannedThisWeek"`
+	RegistriesScannedPrevWeek           int                 `json:"registriesScannedPrevWeek" bson:"registriesScannedPrevWeek"`
+	LinkToRegistriesScanningFiltered    string              `json:"linkToRegistriesScanningFiltered" bson:"linkToRegistriesScanningFiltered"`
+	Top5FailedControls                  []TopCtrlItem       `json:"top5FailedControls" bson:"top5FailedControls"`
+	Top5FailedCVEs                      []TopVulItem        `json:"top5FailedCVEs" bson:"top5FailedCVEs"`
+	ClustersScanned                     []ClusterScanned    `json:"clustersScanned" bson:"clustersScanned"`
```

Comment:
Can't it be just an array of  '''ClusterResourceScanned'''
For what do we need the ClusterScanned type?

Same question about RepositoryScanned  & RegistryScanned

Reply:
I made an external type for registry and repository so that they won't have the same type, and it would be clearer. But I will change it (same was for cluster then I added full and short name)